src/devices/namron.ts

```diff
@@ -69,7 +70,82 @@ const fzLocal = {
         },
     } satisfies Fz.Converter<"hvacThermostat", undefined, ["attributeReport", "readResponse"]>,
 };
+// Namron Simplify 3-button remote (4512793 / 4512794)
+const NAMRON_SIMPLIFY_ACTIONS: Record<number, "press" | "release" | "hold"> = {
+    0: "press",
+    1: "release",
+    2: "hold",
+};
+const HOLD_KEY_SIMPLIFY = "namron_simplify_lastHold";
+const simplify_col = (n: number) => Math.floor((n - 1) / 2) + 1;
+const simplify_sub = (n: number) => (n % 2 === 1 ? "up" : "down");
+
+const fzNamronSimplifyRemote = {
+    cluster: "zosungIRControl",
+    type: ["raw"] as const,
+    convert(
+        model: unknown,
+        msg: unknown,
+        publish: (data: Record<string, unknown>) => void,
+        options: Record<string, unknown>,
+        meta: Record<string, unknown>,
+    ) {
+        // ---- Byte parsing uten any ----
+        type RawContainer = {data?: number[]};
+        type MaybeData = {data?: RawContainer | number[] | Record<string, number>; type?: string};
+
+        const container = msg as MaybeData;
+        const d = container.data;
+        let bytes: number[] = [];
+
+        if (container.type === "raw" && d && typeof d === "object" && "data" in (d as RawContainer) && Array.isArray((d as RawContainer).data)) {
+            bytes = (d as RawContainer).data as number[];
+        } else if (Array.isArray(d)) {
+            bytes = d as number[];
+        } else if (d && typeof d === "object") {
+            const obj = d as Record<string, number>;
+            const keys = Object.keys(obj)
+                .filter((k) => !Number.isNaN(Number(k)))
+                .sort((a, b) => Number(a) - Number(b));
+            bytes = keys.map((k) => obj[k]);
+        }
+
+        if (bytes.length === 0) return;
+
+        const btn = bytes.at(-2);
+        const raw = bytes.at(-1);
+        if (btn == null || raw == null) return;
+
+        const kind = NAMRON_SIMPLIFY_ACTIONS[raw as 0x00 | 0x01 | 0x02];
+        const base = `button_${simplify_col(btn)}_${simplify_sub(btn)}_`;
+
+        if (!kind) {
+            const lastHold = store.getValue(meta.device as never, HOLD_KEY_SIMPLIFY) as string | undefined;
+            if (lastHold?.endsWith("_hold")) {
+                publish({action: lastHold.replace("_hold", "_release")});
+                store.putValue(meta.device as never, HOLD_KEY_SIMPLIFY, null);
+            }
+            return;
+        }
+
+        if (kind === "hold") {
+            store.putValue(meta.device as never, HOLD_KEY_SIMPLIFY, `${base}hold`);
+            publish({action: `${base}hold`});
+            return;
+        }
 
+        if (kind === "release") {
+            publish({action: `${base}press`});
+            publish({action: `${base}release`});
+            return;
+        }
+
+        publish({action: `${base}press`});
+    },
+    // biome-ignore lint/suspicious/noExplicitAny: required generic cast for Converter
+} as unknown as Fz.Converter<any, any, any>;
```

Comment:
Could you prevent the use of this nasty casting? I think it can be done with a custom cluster, recent example: https://github.com/Koenkk/zigbee-herdsman-converters/pull/10745/files